crates/hir-ty/src/tests/simple.rs

```diff
@@ -3011,14 +3011,14 @@ struct TS(usize);
 fn main() {
     let x;
     [x,] = &[1,];
-  //^^^^expected &[i32; 1], got [{unknown}; _]
+  //^^^^expected &[i32; 1], got [{unknown}; {unknown}]
```

Comment:
I think I'd prefer to keep the `_` for unknown consts?

Reply:
@flodiebold why so? I've thought it would be more consistent, as we already have `{unknown}` for unknown types? Why make consts different?

Reply:
Yeah, I'm not sure about the one for types either, but unknown consts will be much more common than unknown types for now, so I don't want to have an `{unknown}` on every array.